code/controllers/subsystem.dm

```diff
@@ -2,33 +2,34 @@
 
 /datum/subsystem
 	// Metadata; you should define these.
-	var/name				//name of the subsystem
-	var/priority = 0		//priority affects order of initialization. Higher priorities are initialized first, lower priorities later. Can be decimal and negative values.
+	var/name = "fire coderbus" //name of the subsystem
```

Comment:
ya, for a while, there was a subsystem with no name, and it looked stupid on the stat panel, so i added this so it looks less stupid next time somebody forgets to name a subsystem